Sources/Commands/Workspace.swift

```diff
@@ -150,6 +150,39 @@ public class Workspace {
         }
     }
 
+    /// A struct representing all the current manifests (root + external) in a package graph.
+    struct DependencyManifests {
+        /// The root manifest.
+        let root: Manifest
+
+        /// The dependency manifests in the transitive closure of root manifest.
+        let dependencies: [Manifest]
+
+        /// Computes the URLs which are declared in the manifests but aren't present in dependencies.
+        var missingURLs: Set<String> {
```

Comment:
I think this should be a function not a property, since it does significant work.